src/main/webapp/data/userMapData.json

```diff
@@ -65,6 +67,7 @@
       "University of New South Wales",
       "University of Queensland",
       "University of South Australia",
+      "University of the Sunshine Coast",
```

Comment:
Let's put this line below `University of Technology, Sydney` to follow alphabetical order.